### ISSUE TITLE
חיווי קולי

### ISSUE THREAD

**Issue by LievreAI** [eladnava/redalert-android#56]

**Comment by eladnava** [comment]
Thanks for the suggestion @LievreAI, it indeed would be a nice addition, I'll look into it.

**Comment by LievreAI** [comment]
יש התקדמות לגבי החיווי הקולה ?

**Comment by eladnava** [comment]
Yes, unfortunately Android does not have any built-in text-to-speech capabilities in Hebrew, so the only alternative is to bundle the app with thousands of sound files for each city / settlement name being read aloud along with the threat type, or to have the app download the sound bundle later.

This is what the [Tzofar](https://play.google.com/store/apps/details?id=com.redalert.tzevaadom) app does.

Since the Tzofar app does this pretty well already, and not many users have asked for this feature, it is currently on low priority. If you really care about having this feature, please use the Tzofar app alongside (or instead of) this one.
